### PR TITLE
Release v0.8.0

### PR DESCRIPTION
**Pull request by MrAlias**
### Added

- Add the `WithPropagator` option to `github.com/signalfx/splunk-otel-go/distro` along with parsing of the `OTEL_PROPAGATORS` environment variable to set the global OpenTelemetry `TextMapPropagator`. (#295)
- Add the `WithTraceExporter` and `WithTLSConfig` options to `github.com/signalfx/splunk-otel-go/distro` along with parsing of the `OTEL_TRACES_EXPORTER` environment variable to set the global OpenTelemetry `SpanExporter` used by the `SDK` to export traces. (#300)
- Add the `splunk.distro.version` attribute to the SDK resource. (#332)
- Add the `WithLogger` option to `github.com/signalfx/splunk-otel-go/distro` along with parsing of the `OTEL_LOG_LEVEL` environment variable to set the logging level of the default logger used. (#336)
- The `SPLUNK_REALM` environment variable is now supported. If set, the exporter will use the corresponding Splunk ingest endpoint. (#725)

### Changed

- The `SDK` from `github.com/signalfx/splunk-otel-go/distro` now uses an OTLP exporter by default. The previous default Jaeger thrift exporter can still be used by setting the `OTEL_TRACES_EXPORTER` environment variable to `jaeger-thrift-splunk`, or by directly passing the user configured exporter with a `WithTraceExporter` option. (#300)
- Update `go.opentelemetry.io/otel*` dependencies from [`v1.3.0`][otel-v1.3.0] to [`v1.6.1`][otel-v1.6.1]. (#656, #720)
- Update `go.opentelemetry.io/contrib*` dependencies from [`v1.4.0`/`v0.30.0`][contrib-v1.4.0] to [`v1.6.0`/`v0.31.0`][contrib-v1.6.0]. (#720)
- The `OTEL_TRACES_SAMPLER` environment variable is now honored instead of only defaulting to an always-on sampler. (#724)
- Set span limits to the Splunk defaults (the link count is limited to 1000, the attribute value length is limited to 12000, and all other limits are set to be unlimited) if they are not set by the user with environment variables. (#723)

### Fixed

- Consistently import `github.com/jackc/pgx/v4`, instead of `github.com/jackc/pgx`, in the `github.com/signalfx/splunk-otel-go/instrumentation/github.com/jackc/pgx/splunkpgx` instrumentation. (#478)

[Unreleased]: https://github.com/signalfx/splunk-otel-go/compare/v0.8.0...HEAD
[0.8.0]: https://github.com/signalfx/splunk-otel-go/releases/tag/v0.8.0
[0.7.0]: https://github.com/signalfx/splunk-otel-go/releases/tag/v0.7.0
[0.6.0]: https://github.com/signalfx/splunk-otel-go/releases/tag/v0.6.0
[0.4.0]: https://github.com/signalfx/splunk-otel-go/releases/tag/v0.4.0
[0.3.0]: https://github.com/signalfx/splunk-otel-go/releases/tag/v0.3.0
[0.2.0]: https://github.com/signalfx/splunk-otel-go/releases/tag/v0.2.0
[0.1.0]: https://github.com/signalfx/splunk-otel-go/releases/tag/v0.1.0

[otel-v1.6.1]: https://github.com/open-telemetry/opentelemetry-go/releases/tag/v1.6.1
[otel-v1.6.0]: https://github.com/open-telemetry/opentelemetry-go/releases/tag/v1.6.0
[otel-v1.3.0]: https://github.com/open-telemetry/opentelemetry-go/releases/tag/v1.3.0
[otel-v1.0.0-RC3]: https://github.com/open-telemetry/opentelemetry-go/releases/tag/v1.0.0-RC3
[otel-v1.0.0-RC2]: https://github.com/open-telemetry/opentelemetry-go/releases/tag/v1.0.0-RC2
[otel-v1.0.0-RC1]: https://github.com/open-telemetry/opentelemetry-go/releases/tag/v1.0.0-RC1
[otel-v0.20.0]: https://github.com/open-telemetry/opentelemetry-go/releases/tag/v0.20.0
[otel-v0.19.0]: https://github.com/open-telemetry/opentelemetry-go/releases/tag/v0.19.0

[contrib-v1.6.0]: https://github.com/open-telemetry/opentelemetry-go-contrib/releases/tag/v1.6.0
[contrib-v1.4.0]: https://github.com/open-telemetry/opentelemetry-go-contrib/releases/tag/v1.4.0
[contrib-v0.28.0]: https://github.com/open-telemetry/opentelemetry-go-contrib/releases/tag/v0.28.0
[contrib-v0.23.0]: https://github.com/open-telemetry/opentelemetry-go-contrib/releases/tag/v0.23.0
[contrib-v0.22.0]: https://github.com/open-telemetry/opentelemetry-go-contrib/releases/tag/v0.22.0
[contrib-v0.21.0]: https://github.com/open-telemetry/opentelemetry-go-contrib/releases/tag/v0.21.0
[contrib-v0.20.0]: https://github.com/open-telemetry/opentelemetry-go-contrib/releases/tag/v0.20.0
[contrib-v0.19.0]: https://github.com/open-telemetry/opentelemetry-go-contrib/releases/tag/v0.19.0

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -8,6 +8,8 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 
 ## [Unreleased]
 
+## [0.8.0] - 2022-04-01
+
```
Comment (theletterf) [inline CHANGELOG.md:12]: @MrAlias Awesome. Could you do a final check of the docs merge request and address Robert's concerns on TLS?

**Review by theletterf — APPROVED** [review]

**Comment by pellared** [comment]
I think this https://github.com/signalfx/splunk-otel-go/pull/492 should be added to the `Changed` section

**Comment by MrAlias** [comment]
> I think this #492 should be added to the `Changed` section MAYBE this one #494 as well (I am not sure)

Those are not user facing changes. I'm not sure they are relevant to the end user reading the changelog.

**Comment by pellared** [comment]
This one is user-facing: https://github.com/signalfx/splunk-otel-go/pull/492

Different package path (user would need to update the `go.mod`)

**Comment by MrAlias** [comment]
> This one is user-facing: #492
> 
> Different package path (user would need to update the `go.mod`)

Gotcha :+1:

File: CHANGELOG.md [hunk 2]
```diff
@@ -35,10 +35,10 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
   environment variable to `jaeger-thrift-splunk`, or by directly passing the
   user configured exporter with a `WithTraceExporter` option. (#300)
 - Update `go.opentelemetry.io/otel*` dependencies from [`v1.3.0`][otel-v1.3.0]
-  to [`v1.6.1`][otel-v1.6.1]. (#656, #720)
+  to [`v1.6.1`][otel-v1.6.1]. (#656, #721)
```
Comment (pellared) [inline CHANGELOG.md:41]:
```suggestion
  to [`v1.6.1`][otel-v1.6.1]. (#406, #471, #656, #721)
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -35,10 +35,10 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
...
 - Update `go.opentelemetry.io/contrib*` dependencies from
-  [`v1.4.0`/`v0.30.0`][contrib-v1.4.0] to [`v1.6.0`/`v0.31.0`][contrib-v1.6.0].
-  (#720)
+  [`v1.3.0`/`v0.28.0`][contrib-v1.3.0] to [`v1.6.0`/`v0.31.0`][contrib-v1.6.0].
+  (#721)
```
Comment (pellared) [inline CHANGELOG.md:44]:
```suggestion
  (#406, #721)
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -8,6 +8,8 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 
 ## [Unreleased]
 
+## [0.8.0] - 2022-04-01
```
Comment (pellared) [inline CHANGELOG.md:11]:
```suggestion
## [0.8.0] - 2022-04-05
```

**Review by pellared — APPROVED** [review]
I left a few suggestions. Everything else LGTM. I will let you know when I finish smoke testing.

**Pull request merged**